forgerock-auth/src/main/java/org/forgerock/android/auth/Result.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024 ForgeRock. All rights reserved.
+ *
+ *  This software may be modified and distributed under the terms
+ *  of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth
+
+/**
+ * A sealed class representing a result, which can be either Success or Failure.
+ *
+ * @param L The type of the success value.
+ * @param R The type of the failure value.
+ */
+internal sealed class Result<out L, out R> {
```

Comment:
this is a built in class in swift for success and failure  , looks like its similat , can we keep the same name L and R instead

public enum Result<Success, Failure>